products/ibm-aix.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IBM AIX
+category: os
+iconSlug: ibm
+permalink: /ibm-aix
+alternate_urls:
+-   aix
+versionCommand: oslevel -s
+releasePolicyLink: https://www.ibm.com/support/pages/standard-and-enhanced-ibm-support-lifecycle-policies
+releaseLabel: "{{'__RELEASE_CYCLE__'|split:'.'|pop|join:'.'}} TL{{'__RELEASE_CYCLE__'|split:'.'|last}}"
+changelogTemplate: "https://www.ibm.com/docs/en/aix/{{'__RELEASE_CYCLE__'|split:'.'|pop|join:'.'}}?topic=notes-aix-{{'__RELEASE_CYCLE__'|replace:'.',''}}-release"
+releaseDateColumn: true
+eolColumn:  <abbr title="End of Service Pack Support">EoSPS</abbr>
```

Comment:
It doesn't make sense to hide such an uncommon abbreviation, better to show it directly:

```suggestion
eolColumn: End of Service Pack Support (<abbr title="End of Service Pack Support">EoSPS</abbr>)
```